website/docs/integrator-section/verifiable-document/ethereum/wallet.md

```diff
@@ -1,16 +1,24 @@
 ---
 id: wallet
-title: Creating a wallet
-sidebar_label: Creating a wallet
+title: Creating a Wallet
+sidebar_label: Creating a Wallet
 ---
 
-The first step that we will need to go through, is the [wallet](/docs/docs-section/appendix/glossary#wallet) creation:
+The first step that we will perform is to create the [wallet](/docs/docs-section/appendix/glossary#wallet). 
+
+After that, you will get the public address of the wallet and be able to verifty the wallet balance. 
+
+If necessary, use ether faucet to top up your wallet.
```

Comment:
I think at this point the user won't know if it is necessary or not. Also considering this is in the `ethereum` section it will definitely be necessary to have some ethers in order to deploy the document store and issue documents.

Should we remove this line and add a note under `## Adding ethers to your wallet` that having some ethers balance will be required for the following steps?

Reply:
Thanks for adding comments, agree.